edx_proctoring/templates/proctored_exam/instructions.html

```diff
@@ -87,27 +136,26 @@
     }
   );
 
-  $(document).ready(function(){
-    _waiting_for_proctored_interval = setInterval(
-      poll_exam_started,
-      5000
-    );
-  });
 
   function poll_exam_started() {
     var url = $('.instructions').data('exam-started-poll-url') + '?sourceid=instructions';
     $.ajax(url).success(function(data){
       if (data.status === 'ready_to_start') {
-        if (_waiting_for_proctored_interval != null) {
-          clearInterval(_waiting_for_proctored_interval)
-        }
         // we've state transitioned, so refresh the page
         // to reflect the new state (which will expose the test)
         location.reload();
       }
+      else {
+          debugger;
+          // The proctoring setup is not yet complete.
+          // Show a modal indicating that the user is not done yet.
+          accessible_error("You have not yet completed the proctoring setup.");
```

Comment:
Can we change the message to: "You must complete the proctoring setup before you can start the exam."